app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  include ControllerHelper
+  before_action :user, only: %i[show edit update destroy]
+
+  def index
+    @users = User.page params[:page]
+  end
+
+  def show; end
+
+  def edit; end
+
+  def destroy
+    @user.destroy
+    redirect_to [:users]
```

Comment:
interesting variation
prefer to write direct route: `users_path`

it's nice to know different things but the simpler your code is, the better it is

Reply:
done